src/components/Checkout.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Elements, StripeProvider } from 'react-stripe-elements';
+import CheckoutForm from './CheckoutForm';
+
+function Checkout({ fireCreateNewCustomer, billingEmail, handleInputChange }) {
+  return (
+    // move this test key into dotenv
+    <StripeProvider apiKey="pk_test_2tIDnmax83LWPMlH2j1eiu9a00CtNJbDfF">
```

Comment:
Does this need to be secure in someway? Like have it in the backend and send to request from the backend?

Reply:
I have no idea how stripe works so feel free to disregard if it doesn't make sense

Reply:
The keys come as a pair, one for the FE is public, and one for BE is private so it's not crazy bad to have this one here but was thinking to move it to a .env anyway at some point.